lib/tapioca/cli.rb

```diff
@@ -257,6 +261,7 @@ def gem(*gems)
       command_args = {
```

Comment:
We need a check right above this line to raise `MalformattedArgumentError` if `--include-dependencies` is used without supplying any `gems`.